site/docs/theming/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Theming
+layout: DetailTechnical
+sidebar:
+  label: Overview
+---
+
+The Salt design system enables teams to modify design variables and theme projects to their requirements. Understanding how Salt structures and manages these design variables will help you create consistent themes for your project.
+
+Below we'll explain what a token is, how different token layers interact, and how this structure makes theming more accessible.
+
+<ExampleContainer type="positive" customPillText="Best practice">
+  At this stage in Salt's development, we recommend you only modify
+  characteristic design tokens when theming your project.
+</ExampleContainer>
+
+## Theming with tokens
+
+Salt's approach to theming enables you to modify design variables, such as color or opacity, in a simple and consistent way.
+
+Salt uses tokens to store design decisions. Components can reuse tokens, meaning they help to define the design rules and aid the cohesiveness of the overall user interface.
+
+Instead of overriding design variables within a single component, we recommend modifying the tokens that the component references. Maintaining this token-based structure offers several benefits:
+
+- Streamlined design changes that automatically apply to related components
+- Increased consistency across all components, improving the overall user experience
+- Simplified, drag-and-drop approach when adding new components or patterns
+- Single reference points for commonly-used values
+
+Note that theming will involve an element of maintenance as future versions of Salt may modify the token list.
+
+## Token tiers
+
+Salt's tokens are organized into three tiers. From most generalized to most specific, the tiers are:
+
+1. Foundational tokens
+2. Palette tokens
+3. Characteristic design tokens
+
+These tokens can reference each other and inherit values from lower tiers.
+
+Take the status characteristic tokens as an example. Components like [`Toast`](/salt/components/toast/examples) use these tokens to denote status using color, such as a green foreground color to denote success.
```

Comment:
What's general rule about caps, e.g. should "status characteristic" being capitalized like component names?